pkg/audit/cursor.go

```diff
@@ -120,19 +91,72 @@ func getRandomStripe(pointer *pb.Pointer) (index int64, err error) {
 		return 0, nil
 	}
 
-	randomStripeIndex, err := rand.Int(rand.Reader, big.NewInt(pointer.GetSegmentSize()/int64(redundancy.StripeSize())))
-	if err != nil {
-		return -1, err
+	var src cryptoSource
+	rnd := rand.New(src)
```

Comment:
nit:  using a crypto random source still seems like overkill to me

Reply:
I also think it is overkill

Reply:
lol

Reply:
here is where I could say that `rnd` isn't a good abbreviation, however it's totally fine to me as it's used in a few lines only and it's clear from where it comes